lib/commits.js

```diff
@@ -1,6 +1,8 @@
 const log = require('./log')
 const paginate = require('./pagination')
-const _ = require('lodash')
+const get = require('lodash/get')
+const uniqBy = require('lodash/uniqBy')
+const flatten = require('lodash/flatten')
```

Comment:
Are these changes required to get it to work or to reduce the compiled output?

Reply:
I am trying to reduce the compiled output here yes.